libs/langchain-google-common/src/connection.ts

```diff
@@ -177,6 +210,8 @@ export abstract class GoogleAIConnection<
 
   client: GoogleAbstractedClient;
 
+  api;  // FIXME: Make this a real type
```

Comment:
Explicitly type as `any`/`unknown`

Reply:
Typed as any.
(I'm expecting that when I do the claude api stuff that I'll give this a real type.)